client/packages/invoices/src/Prescriptions/DetailView/columns.ts

```diff
@@ -73,18 +73,22 @@ export const usePrescriptionColumn = ({
         accessor: ({ rowData }) => {
           if ('lines' in rowData) {
             const { lines } = rowData;
-            const noteSections = lines
-              .map(({ batch, note }) => ({
-                header: batch ?? '',
-                body: note ?? '',
-              }))
-              .filter(({ body }) => !!body);
+            if (!lines) return null;
+
+            // All the lines should have the same note, so we just take the first one
+            const lineWithNote = lines.find(({ note }) => !!note);
+            if (!lineWithNote) return null;
+
+            const noteSections = [
+              {
+                header: null,
+                body: lineWithNote.note ?? '',
+              },
+            ];
+
             return noteSections.length ? noteSections : null;
-          } else {
-            return rowData.batch && rowData.note
```

Comment:
Don't think we need this for prescriptions maybe makes sense for outbound shipments? Not sure...